portfolio/src/main/webapp/greeting.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Uses arrow functions to fetch and display greeting from the data servlet.
+ */
+function getFirstGreeting() {
+
+	fetch('/data').then(response => response.json()).then((greetings) => {
+    // greetings is an object, not a string, so we have to reference its fields to create HTML content
```

Comment:
Nit: Align comment indentation with the code lines below.

Reply:
Got it. I have made the correction